diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8b960402..37fac11d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -326,12 +326,6 @@ add_subdirectory(testing)
 # CDK
 #
 
-add_subdirectory(cdk)
-
-foreach(T cdk cdk_foundation cdk_mysqlx cdk_proto_mysqlx cdk_parser)
-  set_target_properties(${T} PROPERTIES FOLDER "CDK")
-endforeach()
-
 #
 # Project's public headers
 #
@@ -344,7 +338,7 @@ INCLUDE_DIRECTORIES(include)
 # Note: it is included before setting higher warning levels.
 #
 
-add_config_option(WITH_JDBC BOOL DEFAULT OFF
+add_config_option(WITH_JDBC BOOL DEFAULT ON
  "Whether to build a variant of connector library which implements legacy JDBC API"
 )
 
@@ -363,6 +357,7 @@ if(WITH_JDBC)
 
 endif()
 
+return()
 
 #
 # Set higher warning level for the main connector code.
diff --git a/doc/doxygen.cfg b/doc/doxygen.cfg
index 7bf32bcf..33d8bbbc 100644
--- a/doc/doxygen.cfg
+++ b/doc/doxygen.cfg
@@ -66,7 +66,7 @@ PROJECT_NAME           = "MySQL Connector/C++"
 # could be handy for archiving the generated documentation or if some version
 # control system is used.
 
-PROJECT_NUMBER         = 8.0.23
+PROJECT_NUMBER         = 9.5.0
 
 # Using the PROJECT_BRIEF tag one can provide an optional one line description
 # for a project that appears at the top of each page and should give viewer a
diff --git a/version.cmake b/version.cmake
index 53f8f540..3f2e37c1 100644
--- a/version.cmake
+++ b/version.cmake
@@ -34,8 +34,8 @@ set(COPYRIGHT_YEAR "2024" CACHE INTERNAL "version info")
 # Connector/C++ version
 #
 
-set(CONCPP_VERSION_MAJOR  8 CACHE INTERNAL "version info")
-set(CONCPP_VERSION_MINOR  4 CACHE INTERNAL "version info")
+set(CONCPP_VERSION_MAJOR  9 CACHE INTERNAL "version info")
+set(CONCPP_VERSION_MINOR  5 CACHE INTERNAL "version info")
 set(CONCPP_VERSION_MICRO  0 CACHE INTERNAL "version info")
 # Level is "-alpha", "-beta", empty if GA
 set(CONCPP_VERSION_LEVEL  "" CACHE INTERNAL "version info")
